Assignments/docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.3"
```

Comment:
overall I'm not sure why you have 2 Mariadb instances. I would think you only need one for what you're doing here. We're definitely going to need more than one for the final, though.